lib/animina_web/plugs/language.ex

```diff
@@ -0,0 +1,57 @@
+defmodule AniminaWeb.Plugs.AcceptLanguage do
+  import Plug.Conn
+
+  @moduledoc """
+  This is a module that creates a plug to intercept the request headers,
+  extracts the *Accept-Language* header and injects it into the user's session
+  """
+
+  def init(options), do: options
+
+  def call(conn, _options) do
+    conn
+    |> extract_accept_language()
+    |> case do
+      [language | _] ->
+        conn |> put_session("language", String.split(language, "-") |> Enum.at(0, "en"))
```

Comment:
I didn't dive into the code but I am missing the `de` part here. Is it missing?

Reply:
I've added some more logic to configure Gettext with the user's language
- `en-whatever` will be parsed to `en`
- `de-whatever` will be parsed to `de`

original `accept-language` header will be saved as is.